modules/administration-guide/pages/configuring-networking.adoc

```diff
@@ -9,6 +9,9 @@
 
 * xref:configuring-network-policies.adoc[]
 * xref:configuring-che-hostname.adoc[]
+ifeval::["{prod-id-short}" == "che"]
+* xref:configuring-che-with-self-signed-certificate.adoc[]
```

Comment:
Here is what you need to do:
1. Add a new file `modules/administration-guide/pages/snip_che-xref-configuring-che-with-self-signed-certificate.adoc` that contains just one line:
```
* xref:configuring-che-with-self-signed-certificate.adoc[]
```
2. In the current file, add an `include` statement on this line:
```suggestion

include::example$snip_{project-context}-xref-configuring-che-with-self-signed-certificate.adoc[]

```

3. Ask @deerskindoll to create a BLANK `snip_devspaces-xref-configuring-che-with-self-signed-certificate.adoc` file in https://gitlab.cee.redhat.com/red-hat-developers-documentation/devspaces-documentation/-/tree/main/modules/administration-guide/examples.